Bottom: 71cbde2dd6f2a685034d542258dd6cd8a40b470a
Top:    afa3c5ff4fac5a3cf145c86914ed679fd2176565
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-20 23:22:00 +0100

hid/common: Draw the hole outline for pins / vias being moved or placed


---

diff --git a/src/crosshair.c b/src/crosshair.c
index f2ad726..e724537 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -249,7 +249,7 @@ XORDrawElement (ElementTypePtr Element, LocationType DX, LocationType DY)
     PinType moved_pin = *pin;
     moved_pin.X += DX; moved_pin.Y += DY;
 
-    gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pin, false, false);
+    gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pin, true, false);
   }
   END_LOOP;
